dashbuilder/dashbuilder-client/dashbuilder-widgets/src/main/java/org/dashbuilder/client/widgets/dataset/editor/prometheus/PrometheusDataSetDefAttributesEditor.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.client.widgets.dataset.editor.prometheus;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import com.google.gwt.editor.client.EditorDelegate;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import org.dashbuilder.client.widgets.resources.i18n.DataSetEditorConstants;
+import org.dashbuilder.common.client.editor.ValueBoxEditor;
+import org.dashbuilder.dataset.client.DataSetClientServices;
+import org.dashbuilder.dataset.def.PrometheusDataSetDef;
+import org.gwtbootstrap3.client.ui.constants.Placement;
+import org.uberfire.client.mvp.UberView;
+
+/**
+ * <p>Prometheus Data Set specific attributes editor presenter.</p>
+ * 
+ */
+@Dependent
+public class PrometheusDataSetDefAttributesEditor implements IsWidget, org.dashbuilder.dataset.client.editor.PrometheusDataSetDefAttributesEditor {
```

Comment:
Can you import `org.dashbuilder.dataset.client.editor.PrometheusDataSetDefAttributesEditor` instead of using its fully qualified name?

Reply:
This specific case I would have name conflict, so I can't use import, had to use FQN